.github/workflows/ci.yml

```diff
@@ -57,6 +85,8 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: olafurpg/setup-scala@v5
```

Comment:
@gabro @olafurpg I noticed this is on v5 not v7. Is that intentional or did it just get missed?

Reply:
just missed, I believe.

Reply:
Updated